s3backgrounddelete/s3backgrounddelete/object_recovery_scheduler.py

```diff
@@ -61,10 +62,79 @@ def isObjectLeakEntryOlderThan(leakRecord, OlderInMins = 15):
         timeDeltaInMns = math.floor(timeDelta.total_seconds()/60)
         return (timeDeltaInMns >= OlderInMins)
 
+    def add_kv_to_msgbus(self, marker = None):
+        """Add object key value to msgbus topic."""
+        self.logger.info("Inside add_kv_to_msgbus.")
+        try:
+            from s3backgrounddelete.object_recovery_msgbus_producer import ObjectRecoveryMsgbusProducer
+
+            if not self.producer:
```

Comment:
Is this check needed? After the fix by foundation team , the sequence on every run will be
1) Get instance of message bus class - Connect can be part of this
2) Setup producer
3) Send

Reply:
Removed.